src/java.desktop/share/classes/java/awt/Font.java

```diff
@@ -1202,7 +1202,7 @@ public Void run() {
             if (cause instanceof FontFormatException) {
                 throw (FontFormatException)cause;
             }
-            throw new IOException("Problem reading font data.");
+            throw new IOException("Problem reading font data.", t);
```

Comment:
Revert this. It is done this way as part of a previous security fix to avoid exposing information.
"    8001038: Resourcefully handle resources "

Reply:
Reverted in [49c6ebd](https://github.com/openjdk/jdk/pull/20169/commits/49c6ebdd4a44f418c380a2fd5f699d7bd65856bc)